datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/AddOwnersModal.tsx

```diff
@@ -149,27 +121,81 @@ export const AddOwnersModal = ({
                 : undefined;
         const displayName = entityRegistry.getDisplayName(result.entity.type, result.entity);
         return (
-            <SearchResultContainer>
-                <SearchResultContent>
-                    <CustomAvatar
-                        size={24}
-                        name={displayName}
-                        photoUrl={avatarUrl}
-                        isGroup={result.entity.type === EntityType.CorpGroup}
-                    />
-                    <div>{displayName}</div>
-                </SearchResultContent>
-            </SearchResultContainer>
+            <Select.Option value={result.entity.urn} key={result.entity.urn}>
```

Comment:
One final thing i want to do: 

The parameter to this function probably should not be a search result.

Instead it should be the entity itself.

Then the value returned by useGetRecommendedOwners should be an Array<Entity> not an Array<SearchResult>

This will make it so that plugging in new recommendation logic is VERY easy since it doesn't need to be a SearchResult

Reply:
Checked and fixed.